docs/_articles/en/user-guide/perf-enhancements.md

```diff
@@ -14,7 +14,7 @@ To enable this beta feature:
 1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
 1. Under Salesforce Feature Previews, select Experimental: Deploy Retrieve.
 
-In this beta release, performance enhancements are effective when you run the **SFDX: Deploy This Source to Org** and **SFDX: Retrieve This Source from Org** command with these metadata types:
+In this beta release, performance enhancements are effective when you run the **SFDX: Deploy This Source to Org** and **SFDX: Retrieve This Source from Org** commands. At this time, the beta enhancements for **SFDX: Retrieve This Source from Org** only works with the following metadata types:
```

Comment:
@ekapner doc update with the change

Reply:
@brpowell - small tweaks (works to work, move the "only", remove "the following" to something more conversational):

At this time, the beta enhancements for **SFDX: Retrieve This Source from Org** work with only these metadata types: